vars/sconsBuild.groovy

```diff
@@ -180,7 +185,7 @@ def call(Map config = [:]) {
     script += 'SCONS_ARGS="' + scons_args + '"\n'
     script += '''# the config cache is unreliable so always force a reconfig
                  # with "--config=force"
-                 if ! scons --config=force $SCONS_ARGS''' +
+                 if ! ''' + scons_exe + ''' --config=force $SCONS_ARGS''' +
```

Comment:
Technically, only need `'` here:

```suggestion
                 if ! ''' + scons_exe + ' --config=force $SCONS_ARGS' +
```

but most certainly not worth pushing a new one just for that.  I'll keep a mental note for the next time I'm in there.